packages/zod/project.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/zod/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
```

Comment:
Also, you have migrate all builds from tsc to rollup. Could we do the same for this package?